src/docs/product/sentry-basics/tracing/trace-view.mdx

```diff
@@ -40,7 +40,7 @@ Also, in these cases you can click "Open In Discover" to see all the events in t
 
 Broken subtraces may be caused by:
 
-- SDK sampling. Setting a sample rate that's too low may prevent the SDK from sending a transaction. [We recommend sending us all of your transaction data](/product/sentry-basics/sampling/).
+- SDK sampling. Setting a sample rate that's too low may prevent the SDK from sending a transaction. We recommend sending us all of your transaction data.
```

Comment:
The best place to link to now is https://docs.sentry.io/product/performance/retention-priorities/#deciding-on-your-sdk-sample-rate

```suggestion
- SDK sampling. Setting a sample rate that's too low may prevent the SDK from sending a transaction. We recommend [sending us all of your transaction data](/product/performance/retention-priorities/#deciding-on-your-sdk-sample-rate).
```